pagerduty/resource_pagerduty_service.go

```diff
@@ -54,16 +54,21 @@ func resourcePagerDutyService() *schema.Resource {
 					"intelligent",
 					"rules",
 				}),
+				Deprecated:    "Use `alert_grouping_parameters.type`",
+				ConflictsWith: []string{"alert_grouping_parameters"},
 			},
 			"alert_grouping_timeout": {
-				Type:     schema.TypeInt,
-				Optional: true,
+				Type:          schema.TypeString,
```

Comment:
`alert_grouping_timeout` must be a string to be able to set it to `"null"` at https://github.com/PagerDuty/terraform-provider-pagerduty/pull/377/files#diff-ef1847adc67b2e7a9ba18b9c2c36d3a602b5b5938936a3841e7e96ed9b62f048L391

Note: this removed line is just moved to https://github.com/PagerDuty/terraform-provider-pagerduty/pull/377/files#diff-ef1847adc67b2e7a9ba18b9c2c36d3a602b5b5938936a3841e7e96ed9b62f048R443